packages/engine/Source/Scene/PointPrimitiveCollection.js

```diff
@@ -497,6 +500,12 @@ function createVAF(context, numberOfPointPrimitives, buffersUsage) {
         componentDatatype: ComponentDatatype.FLOAT,
         usage: buffersUsage[DISTANCE_DISPLAY_CONDITION_INDEX],
       },
+      {
+        index: attributeLocations.splitDirection,
```

Comment:
To save on the number of attributes, which is limited, let's pack this single float component along with `distanceDisplayConditionAndDisableDepth` as a 4th component.

Reply:
Thank you for the comment. 
Fixed, added splitDirection as the 4th component in distanceDisplayConditionAndDisableDepth.